### ISSUE TITLE
Publish new gem with typo fix

### ISSUE THREAD

**Issue by robertlabrie** [tourlane/dynalock#5]
Hi, you fixed the typo in the method name but didn't publish a new gem. Possible to push a 2.0.1?

```
➜  code ruby lock.rb
lock.rb:5:in `<main>': undefined method `acquire_lock' for main:Object (NoMethodError)
Did you mean?  adquire_lock
➜  code gem list | grep dyna
aws-sdk-dynamodb (1.3.0)
aws-sdk-dynamodbstreams (1.0.0)
dynalock (0.2.0)
```

**Comment by damireh** [comment]
Just released and pushed a new version [0.2.1](https://rubygems.org/gems/dynalock/versions/0.2.1)

**Comment by robertlabrie** [comment]
Got it, TYVM. Found you by googling "ruby distributed lock dynamodb" and will be using this gem for my project.
